src/get-started/java-binding.md

```diff
@@ -1404,7 +1409,7 @@ See [sample-config.yml][launcher-sample-config] for the final state of the
 configuration file.
 
 [exonum-launcher]: https://github.com/exonum/exonum-launcher
-[plugins-readme]: https://github.com/exonum/exonum-java-binding/blob/ejb/v0.9.0-rc2/exonum-java-binding/exonum_launcher_java_plugins/README.md
+[plugins-readme]: https://pypi.org/project/exonum-launcher-java-plugins/
```

Comment:
Does it have a compatibility table, if we use unversioned links (with plugins that is easier).

Reply:
I've thought about it, we can use versioned link